deployments/deploy-ssh.sh

```diff
@@ -77,6 +77,10 @@ if [ -z "${PROVISIONER}" ]; then
     PROVISIONER="portworx"
 fi
 
+if [ -z "${PX_NAMESPACE}" ]; then
+    PX_NAMESPACE="kube-system"
+fi
+
```

Comment:
you define default namespace "kube-system" twice. Here and later in tests/common.go defaultPxNamespace                    = "kube-system". Use another login here if PX_NAMESPACE is exists then use it, if not, leave it blank and torpedo init will initialize it as defaultPxNamespace